ios/FluentUI/List/ListItem.swift

```diff
@@ -281,8 +281,24 @@ public struct ListItem<LeadingContent: View,
 
     // MARK: Private variables
 
+    /// The background styling of the `ListItem`.
+    private var backgroundStyleType: ListItemBackgroundStyleType {
```

Comment:
I think we generally try to avoid early returns unless it's a `guard...else` to validate parameters. Maybe this could just be:

```
let styleType: ListItemBackgroundStyleType
if let customBackgroundStyleType {
    styleType = customBackgroundStyleType
} else {
    switch listStyle {
    case .plain, .inset:
        styleType = .plain
    case .insetGrouped:
        styleType = .grouped
    }
}
return styleType
```

https://microsoft.github.io/swift-guide/EarlyReturn.html